src/azul/health.py

```diff
@@ -202,3 +227,45 @@ def response(self):
     def generate_cache(self):
         health_object = dict(time=time.time(), health=self.health.as_json())
         self.storage_service.put(object_key=f'health/{self.health.lambda_name}', data=json.dumps(health_object))
+
+    def archive_fail_messages(self):
+        client = boto3.client('dynamodb')
+        fail_queue = boto3.resource('sqs').get_queue_by_name(QueueName=config.fail_queue_name)
+        while True:
```

Comment:
… we should make sure that this loop doesn't exceed it. The lambda context exposes the remaining runtime. Check out `LambdaContext` in `types.py`.